From bca3793bdd5b8fd4747da39b79259d00fe147cb3 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Mon, 18 Sep 2023 10:36:12 +0100
Subject: [PATCH] pyproject.toml: Move nose from dependencies to
 dev-dependencies

This is only needed when running the build-time tests, and is not needed
for ordinary use of the tool.

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 pyproject.toml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 502ad3e..5953fd8 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -8,11 +8,11 @@ authors = ["Artem Bityutskiy <dedekind1@gmail.com>"]
 [tool.poetry.dependencies]
 python = "^3.8"
 six = "^1.16.0"
-nose = "^1.3.7"
 gpg = "^1.10.0"
 
 [tool.poetry.dev-dependencies]
 black = "^22.3.0"
+nose = "^1.3.7"
 
 [tool.poetry.scripts]
 bmaptool = "bmaptools.CLI:main"
